### ISSUE TITLE
tcp problem

### ISSUE THREAD

**Issue by badbruce07** [ckan/ckan#2276]
Hey goood morning,
I'm having an issue with tcp when attempting to run ckan/ckan.

It appears like this when running netstat:

![screenshot from 2015-02-09 10 45 35](https://cloud.githubusercontent.com/assets/1694078/6109462/e17c9f82-b048-11e4-87f5-14b78bae69ae.png)

How can I resolve such a problem?

**Comment by badbruce07** [comment]
Also when I ran `sudo docker ps` it appears as this:

![screenshot from 2015-02-09 10 54 26](https://cloud.githubusercontent.com/assets/1694078/6109627/2da8cc9a-b04a-11e4-83f0-c933e4192050.png)

Also when I attempt to run one of the container_id for ckan/ckan:latest it keeps telling me address is in use. I tried using `sudo docker stop "container_id#"` to stop but it hasn't. How do I resolve such a problem?
